docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -90,10 +89,37 @@ To update the metadata and playback information on the server side, and to retri
         Tizen.Log.info(LogTag, $"Playback state is {controller.GetPlaybackState()}");
         ```
 
-<a name="send_media"></a>
+## Updating and Retrieving Playlist
+
+To create a playlist from the server side, create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
```

Comment:
@hsgwon Based on this comment: https://github.com/Samsung/tizen-docs/pull/758#discussion_r299394534 from Ms. @mijin85cho, please apply language identifier label for the language type, which is 'csharp' for the code blocks on this page. 
To find language identifier labels, kindly refer to: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#inline-code-blocks-with-language-identifier

Reply:
Done